CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ in `task_utils`
 ### Changed
 - Changed default value of `on_error` parameter in `BCPTask` and `on_bcp_error` parameter in `ADLSToAzureSQL` 
 and `DuckDBToSQLServer` to `fail`. 
+- Added new column into Data Frames created with `Mindful`.
+- Added region parameter as an entry argument in `MindfulToADLS`
+- Changed `test_minful` file to be able to work with new characteristics.
```

Comment:
I'm not sure if we should add information about tests here. You can delete this line.

Reply:
removed!